src/main/java/com/artipie/repo/FlatLayout.java

```diff
@@ -103,13 +104,16 @@ public Slice resolve(final String path) {
                         final Slice slice;
                         if (exist) {
                             slice = new AsyncSlice(
-                                storage.value(key).thenCombine(
+                                StorageAliases.find(storage, key.parent().get()).thenCompose(
```

Comment:
@g4s8 It seems that we should to think about caching. We do a lot of config readings on each request. Maybe create the issue?